.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+
+-- Firstly, thanks for adding this feature! Secondly, please check the key steps against the checklist below to make your contribution easy to merge. --
+
+### Pull Request checklist
+- [ ] This PR includes test cases to demonstrate any code changes. Including one or more of the following:

```

Comment:
Nitpick: I have sort of a "process" question. We recently added a similar PR template at work with checkbox items. It looked good to me, but one person pointed out that in the GitHub page that lists all open PRs, for each one, it shows how many checkboxes there are and how many are checked. Most PRs won't have all the boxes checked (like if that step is not applicable for their PR).

TBH, I was okay with this, but as a compromise, we ended up removing some of the Markdown checkboxes and instead asking people to copy/paste a ✅ character next to the things they'd done.

Just wanted your thoughts on this, definitely not a blocker.

Reply:
Yeah it will say 1 out of 4 items. Or something like this. I can see how that might be confusing.

To be honest I'd be OK just removing the tick boxes and have justing this as a list to remind people, rather than something they have to explicitly tick off. I only included them as they were already in the existing - mostly used - templates. It's up to the reviewer to check they've done the appropriate steps (coupled with the automatic checks like no drops in cover), so not like the tick box does anything.

WDYT?

Reply:
Changed it to a single checkbox so there is still an action on the user to explicitly state they've read the T&Cs :-)